octodns/provider/dnsimple.py

```diff
@@ -30,13 +30,16 @@ def __init__(self):
 
 
 class DnsimpleClient(object):
-    BASE = 'https://api.dnsimple.com/v2/'
 
-    def __init__(self, token, account):
+    def __init__(self, token, account, sandbox):
         self.account = account
         sess = Session()
         sess.headers.update({'Authorization': 'Bearer {}'.format(token)})
         self._sess = sess
+        if sandbox:
+            self.BASE = 'https://api.sandbox.dnsimple.com/v2/'
+        else:
+            self.BASE = 'https://api.dnsimple.com/v2/'
```

Comment:
Now that this is an instance variable rather than a constant please `s/BASE/base/g` so that things match up.